.setup/INSTALL_SUBMITTY_HELPER_SITE.sh

```diff
@@ -4,11 +4,15 @@
 ################################################################################################################
 # COPY THE 1.0 Grading Website
 
+start=$SECONDS
```

Comment:
What does any of this have to do with moving core logic from view to controller for the home page? I think a change this big should have it's own issue and PR.